### PR TITLE
Set Js Exercises as last course of the JS topic

### PR DESCRIPTION
**Pull request by Stefan-Stojanovic**

### CODE REVIEW

**Comment by enki-hq** [comment]
### Changelog

| File Name | Error Message |
|:-:|:-:|
| Pull Request | Couldn't find a CHANGELOG entry for this pull request. |

> If you notice anything wrong with these rules contact andrei@enki.com.

**Comment by enki-hq** [comment]
### Course

| Course Name | Error Message |
|:-:|:-:|
| javascript-playground-questions | Course "next" does not reference an existing course. |
### Changelog

| File Name | Error Message |
|:-:|:-:|
| Pull Request | Couldn't find a CHANGELOG entry for this pull request. |

> If you notice anything wrong with these rules contact andrei@enki.com.

**Comment by enki-hq** [comment]
### Changelog

| File Name | Error Message |
|:-:|:-:|
| Pull Request | Couldn't find a CHANGELOG entry for this pull request. |

> If you notice anything wrong with these rules contact andrei@enki.com.

File: javascript/javascript-playground-questions/README.md [hunk 1]
```diff
@@ -35,3 +35,5 @@ sections:
     - js-beginning-and-end-pairs
     - js-harmonic-series
     
+next:
+  - javascript:react
```
Comment (kapnobatai137) [inline javascript/javascript-playground-questions/README.md:39]: we can remove this `next` here, the js topic doesn't have a react course anymore (it used to have one in the past)
```suggestion
```

**Review by kapnobatai137 — APPROVED** [review]

**Review by kapnobatai137 — APPROVED** [review]

**Pull request merged**